pkg/tester/data.go

```diff
@@ -218,6 +218,17 @@ func InitializeData(
 		return nil, fmt.Errorf("%s: unable to get previously seen accounts", err.Error())
 	}
 
+	// We need to mark the bootstrapped accounts as "seen",
+	// otherwise they won't be reconciled until rosetta-cli restart.
+	if len(config.Data.BootstrapBalances) > 0 {
```

Comment:
Bootstrap balance should be only used when historical balance is disabled, we may not leave this logic in the main code path

Reply:
Thank you!

If the `check:data` configuration file does not contain a bootstrap balances file (as it should be the case for blockchains with historical balances enabled, if I understood correctly), then the newly added behavior is inactivated.

However, since historical balances lookup support is always desired and recommended, this PR can be closed, perhaps?